.circleci/config.yml

```diff
@@ -41,15 +41,14 @@ jobs:
         destination: /build
 
   docker_hub_master:
-    executor: golang
+    machine:
+      image: circleci/classic:201808-01
 
     steps:
     - checkout
-    - setup_remote_docker
```

Comment:
Dropping setup_remote_docker means the docker builds now run locally in the primary container. I don't think this has any downsides beside maybe worse caching. But not sure and something to keep in mind.

Reply:
As far as I know caching is anyway a paid opt-in feature and would require setting `docker_layer_caching: true` as described here https://circleci.com/docs/2.0/building-docker-images/

With the prometheus docker images only being small wrappers around the already built go binaries based on ~5mb busybox, I don't think re-pulling and re-building on every run is really a concern here.

Reply:
We should switch this back to remote_docker, like we did in busybox: https://github.com/prometheus/busybox/pull/20

Reply:
Done